src/doc/DocAnnotator.js

```diff
@@ -871,21 +871,46 @@ class DocAnnotator extends Annotator {
         }
 
         const lastRange = selection.getRangeAt(selection.rangeCount - 1);
-        const rects = lastRange.getClientRects();
-        if (rects.length === 0) {
-            return;
+        const { endContainer, endOffset } = lastRange;
```

Comment:
is it possible to move this into a docUtil method that returns the [x, y] coordinates?

Reply:
Good idea!